Add note about deposit contract diff

dapplion opened the pull request.

dapplion committed:
Add note about deposit contract diff

dapplion committed:
Update script

filoozom commented on beacon_chain.md:
```diff
@@ -10,7 +10,11 @@ Note that modified preset values will result in different SSZ data structures, s
-### Preset diff
+## Deposit contract diff
+
+Gnosis Beacon Chain deposit contract uses an ERC20 token to stake, specifically GNO [`0x9C58BAcC331c9aa871AFD802DB6379a98e80CEdb`](https://gnosisscan.io/token/0x9C58BAcC331c9aa871AFD802DB6379a98e80CEdb). The minimum deposit required is 1 GNO. To match the Beacon Chain's minimum deposit of 32e9 GWei, the deposit contract multiplies by a [factor of 32](https://github.com/gnosischain/deposit-contrac ...
```
> The minimum deposit required is 1 GNO

This is slightly confusing. Technically the smallest deposit is 1 "mGNO", and thus 0.03125 GNO. And then add something about the activation balance being 1 GNO / 32 "mGNO"?

In general maybe we should keep the `mGNO` naming convention (or rename it do something more explicit, `beaconGNO`?), define that somewhere and not talk too much about it in general because it's mainly implementation detail.

dapplion committed:
Update Gnosis Beacon Chain deposit contract details

Clarified the use of mGNO in the Gnosis Beacon Chain and adjusted the explanation of deposit amounts.
